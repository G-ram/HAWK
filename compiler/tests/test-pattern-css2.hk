BEGIN{}
[@#catlinks@]{
  print(this);
}
END{}
