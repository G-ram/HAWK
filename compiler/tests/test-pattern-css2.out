innerhtml=<div id="mw-normal-catlinks">
 <a href="/Special:Categories" title="Special:Categories">Category</a>:
 <ul>
  <li><a href="/Category:Miscellaneous_people" title="Category:Miscellaneous people">Miscellaneous people</a></li>
 </ul>
</div>
nextChild=mw-normal-catlinks
class=catlinks
id=catlinks
